screens/historyHelpers.js

```diff
@@ -0,0 +1,41 @@
+import BASE from '../lib/common';
+
```

Comment:
I also moved this function into this folder, because I anticipated needing more helper functions before realizing we could use `filterByFormula` .. for now I think it's okay though; our other screens are slowly moving functions out as well